### DISCUSSION TITLE
no tag in docker registry

### DISCUSSION THREAD

**Discussion by dufguix** [plausible/analytics#2512] [Self-Hosted Support] [answered]
Hi, I'm trying to start using plausible but it seems there is no tag for plausible/analytics image on Docker Hub.

Is it an issue or is there another registry ?

Thanks. Have a good day.

**Comment by ruslandoga** [comment, marked as answer]
👋 @dufguix 

I see several tags on https://hub.docker.com/r/plausible/analytics/tags, right now the latest stable is `v1.5.1`.

```console
$ docker pull plausible/analytics
Using default tag: latest
latest: Pulling from plausible/analytics
80fa7f07ec7b: Already exists
fc91451d599b: Pull complete
4bacb94e6f3c: Pull complete
807cac6169fb: Pull complete
e9ce7281d43d: Pull complete
d04187b3886a: Pull complete
c0c3c73ca1ef: Pull complete
4f4fb700ef54: Pull complete
Digest: sha256:d8f8760b8b66b0430aeb27a8ddf4c29c96f11855f8f248e8ef2d02a8787eafc6
Status: Downloaded newer image for plausible/analytics:latest
docker.io/plausible/analytics:latest
```

```console
$ docker pull plausible/analytics:v1.5.1
v1.5.1: Pulling from plausible/analytics
Digest: sha256:d8f8760b8b66b0430aeb27a8ddf4c29c96f11855f8f248e8ef2d02a8787eafc6
Status: Downloaded newer image for plausible/analytics:v1.5.1
docker.io/plausible/analytics:v1.5.1
```

**Reply by dufguix** [reply]
Strange.

Firefox show me (even with cleared cache):
Tags not retrieved. An error occurred while loading the tags. Try reloading the page.

But it's fine with chrome.

Thanks.

**Reply by ukutaht** [reply]
Firefox has an adblocker enabled by default which for some reason blocks loading tags from dockerhub. To confirm:
1. Can you see tags on other dockerhub repos? (expect NO)
2. Can you see them if you turn off your adblocker? (expect YES)

**Reply by dufguix** [reply]
In answer to your first question: Yes
I tested pages below and I can see tags. 
https://hub.docker.com/_/nginx/tags
https://hub.docker.com/_/nextcloud/tags
https://hub.docker.com/r/homeassistant/home-assistant/tags


If I turn off uBlock Origin plugin. Indeed, I can see tags on plausible/analytics.

**Reply by ukutaht** [reply]
Ah right, I think it's specific to our dockerhub because the URL contains 'analytics'.

**Reply by dufguix** [reply]
Yes. https://hub.docker.com/r/neo4jbuildservice/analytics/tags has the same behavior.
